### PR TITLE
Remove fluxsvc

### PR DESCRIPTION
**Pull request by sambooo**
This shall no longer live here

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,16 +34,11 @@ build/.%.done: docker/Dockerfile.%
 	touch $@
 
 build/.flux.done: build/fluxd build/kubectl
-build/.flux-service.done: build/fluxsvc build/migrations.tar
 
-build/fluxd: $(FLUXD_DEPS)
+build/fluxd: vendor/
```
Comment (squaremo) [inline Makefile]: What was wrong with `$(FLUXD_DEPS)`?

Comment (sambooo) [reply Makefile]: It called godeps: `FLUXD_DEPS:=$(call godeps,./cmd/fluxd)`. Something I didn't notice when I did that particular migration.

Comment (squaremo) [reply Makefile]: Oh is `godeps` not a thing any more

Comment (sambooo) [reply Makefile]: Wait, yep I'm wrong. Derp.

Comment (squaremo) [reply Makefile]: Do we vendor the other bits and pieces into vendor/ now?

Comment (sambooo) [reply Makefile]: Which other bits and pieces are you referring to?

**Comment by squaremo** [comment]
OMG OMG OMG :100:

**Pull request merged**